sys/net/routing/rpl/Makefile

```diff
@@ -1,8 +1,16 @@
-MODE = $(shell grep "RPL_DEFAULT_MOP RPL_NON_STORING_MODE" ../../include/rpl/rpl_config.h)
-ifneq (,$(MODE))
-	DIRS += rpl_nonstoring
+MODE = $(shell grep "\#define $(RPL_MOP)" ../../include/rpl/rpl_config.h | sed 's/\#define \s*//g')
```

Comment:
I agree that it's more user friendly now, but the numbers are reserved by IANA (and thus unlikely to ever be changed) and stated in the [RFC](https://tools.ietf.org/html/rfc6550#page-129) where everyone can look it up.

But how about making the mapping name-to-number within the Makefile?